tasks/kernel_matrix_testing/README.md

```diff
@@ -1,425 +1,564 @@
-A `stack` may be:
-
-- Created
-- Configured
-- Launched
-- Paused
-- Resumed
-- Destroyed
-
-> All subsequent commands are assumed to be executed from the root directory of the datadog-agent repository.
+> All commands are assumed to be executed from the root directory of the `datadog-agent` repository.
 
 ## Dependencies
 
-All dependencies are installed by the `kmt.init` command, as defined [below](#initializing-the-environment).
+All dependencies are installed by the `kmt.init` command. See [kmt.init](#kmtinit) for details.
 
-> For macOS users: Internet Sharing might need to be enabled for the networking to work properly. Enable it in System Settings -> General -> Sharing if you find problems with the VM networks. It does not matter which interface you enable it on, as long as it is enabled and the connection being shared is the one you use for Internet connection. We'd also appreciate it if you reported it to the eBPF platform team, as it's not clear still whether Internet Sharing needs to be enabled at some point or not.
+For macOS users, see the [macOS Network Issues](#macos-network-issues) section in troubleshooting.
 
-## Getting started
+## Quick Start
 
-A straightforward flow to setup a collections of VMs is as follows:
+### Quick Start for Local VMs
 
-### Initializing the environment
+This guide shows a minimal workflow to get a single local VM running.
 
-This will download all the resources required to launch the VMs, and install all the system dependencies.
+**1. Initialize KMT** (only needs to be done once)
 
-> This step should be done only once.
+This will download required resources and install system dependencies.
 
 ```bash
-# Initialize with specific VM images
-dda inv -e kmt.init --images=ubuntu_22.04,debian_10
+# Initialize with a specific VM image
+dda inv -e kmt.init --images=ubuntu_22.04
 
-# Initialize and download all available VM images for your architecture
+# Or download all available VM images for your architecture (this can be slow)
 dda inv -e kmt.init --all-images
-
-# If you intend to use KMT *only* for managing remote VMs
-dda inv -e kmt.init --remote-setup-only
 ```
 
-The `--images` parameter is required unless `--remote-setup-only` or `--all-images` is specified. You can use `dda inv -e kmt.ls` to see available images.
+**2. Create and Configure a Stack**
 
-This command will also ask you for the default SSH key to use, so it does not need to be provided every time. You can configure the SSH key again at any time running `dda inv -e kmt.config-ssh-key`. See more details below
+A stack is a collection of VMs. This command will create a new stack and configure it to run one local Ubuntu 22.04 VM.
 
-#### SSH key configuration
+```bash
+# --init-stack creates the stack if it does not exist
+# If --stack is not provided, a name is generated from your git branch
```

Comment:
I think the **Quick Start** should not include the `--stack` parameter. The pattern we want to encourage is to rely on KMT to automatically generate stack names based on git branches.

Explicitly specifying the stack name is a "power user" feature.